### PR TITLE
feat(about): add about section

### LINKED ISSUE #8: Create the About page.
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
The design should be the same and responsive.

**Screenshots of the design/if any**
image

**Additional context**

### PR DESCRIPTION
**Pull request by RawandDev**

### CODE REVIEW

**Comment by RawandDev** [comment]
don't accept until everyone sends their information please.

File: src/components/About/TeamCard.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        </h3>
+      </div>
+      <p>{member.description}</p>
+    </div>
+  ));
+
+  return <>{teamMember}</>;
+}
+
+export default TeamCard;
```
Comment (MuhannedNoman) [inline src/components/About/TeamCard.jsx]: How about you make this like 
```
const TeamMember = ({image, name, stuff}) => {
 return (
  <div>
    <image src="" />
    <h1> {Name} </h1>
<p>{stuff} </p>
</div
)
}
```

Then in About.jsx page you map through those and pass the data.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]

File: src/components/About/TeamCard.jsx (hunk 1 above)
Comment (RawandDev) [reply src/components/About/TeamCard.jsx]: I have made some changes. Please let me know if that is what you expected.

Comment (RawandDev) [reply src/components/About/TeamCard.jsx]: Another question please. After solving conflicts there are two `About.jsx`. One is in the `pages` and the other is in the `components`. Should I copy the content of the `component/About.jsx` and add it inside `pages/About.jsx` then remove the `About.jsx` inside `components`?

File: src/pages/About/TeamCard.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
```
Comment (MuhannedNoman) [inline src/pages/About/TeamCard.jsx]: Let's move this to the components folder.

File: src/pages/About/About.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            Libvery
+          </span>{" "}
+          to help you easily find the books you need. Our aim is to make life
+          much more easier to access books and make a better generation.
+        </p>
+      </div>
+      <div className="text-center mx-10 sm:mx-32 lg:mx-52 mt-10">
+        <h1 className="text-2xl font-semibold mb-3">Our Team</h1>
+        {teamMembers.map(({ name, image, position, description }) => {
+          // const { name, image, position, description } = member;
```
Comment (MuhannedNoman) [inline src/pages/About/About.jsx]: Remove comment.

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
LGTM just added a few comments to make things better. Feel free to merge after implementing them.

**Comment by RawandDev** [comment]
> LGTM just added a few comments to make things better. Feel free to merge after implementing them.

All done. Thank you.

**Pull request merged**